skills/csharp/experimental/hospitalityskill/Dialogs/MainDialog.cs

```diff
@@ -204,7 +204,9 @@ private async Task<DialogTurnResult> IntroStepAsync(WaterfallStepContext stepCon
                 // If bot is in local mode, prompt with intro or continuation message
                 var promptOptions = new PromptOptions
                 {
-                    Prompt = stepContext.Options as Activity ?? _templateManager.GenerateActivity(MainResponses.WelcomeMessage)
+                    Prompt = stepContext.Options as Activity ?? _templateManager.GenerateActivity(
```

Comment:
does this apply to other skills?

Reply:
no. and it will be updated in latest change